packages/components/tests/dummy/app/templates/components/table.hbs

```diff
@@ -102,6 +102,9 @@
         <li>tall</li>
       </ol>
     </dd>
+    <dt>caption <code>string</code></dt>
+    <dd><p>Adds a caption for users with assistive technology. If set on a sortable table, the provided table caption is
```

Comment:
Is it worth making it clear that setting this value will not result in a visible message? It may be inferred from the first statement, but not sure.

Reply:
@alex-ju maybe "adds a non-visible caption for users with assistive technology" ? Would that work for you?

Reply:
yes, that sounds great!